pkg/report/report.go

```diff
@@ -21,7 +23,36 @@ type ScanReportParser interface {
 	Parse(string) (*types.UpdateManifest, error)
 }
 
-func TryParseScanReport(file string) (*types.UpdateManifest, error) {
+func TryParseScanReport(file, scanner string) (*types.UpdateManifest, error) {
+
+	if scanner == "" {
```

Comment:
you'll need to handle `trivy` case

Reply:
there is also a `QualysParser`, do we want it to remain as default like trivy (like it is now), or it should be removed and added as a plugin?

Reply:
I am okay to remove to qualys as we don't have access to it anymore